### PR TITLE
Add portuguese translation

### PR DESCRIPTION
**Pull request by 00tiagopolicarpo00**
Add portuguese translation

### CODE REVIEW

File: vocab/pt-pt/PrevKeyword.voc [hunk 1]
```diff
@@ -0,0 +1 @@
+anterior faixa
```
Comment (augustnmonteiro) [inline vocab/pt-pt/PrevKeyword.voc:1]: in this case will be `faixa anterior` but I'd add `musica anterior` too

**Review by augustnmonteiro — CHANGES_REQUESTED** [review]

File: vocab/pt-pt/NextKeyword.voc [hunk 2]
```diff
@@ -0,0 +1,2 @@
+próxima faixa
```
Comment (augustnmonteiro) [inline vocab/pt-pt/NextKeyword.voc:1]: I'd add `proxima musica` too

File: vocab/pt-pt/CurrentlyPlayingKeyword.voc [hunk 3]
```diff
@@ -0,0 +1,4 @@
+o que está actualmente reproduzir 
+o que você está reproduzir
+o que estamos ouvindo
+o que está reproduzir
```
Comment (augustnmonteiro) [inline vocab/pt-pt/CurrentlyPlayingKeyword.voc:4]: Add `o que estou ouvindo` too

File: vocab/pt-pt/ResumeKeyword.voc [hunk 4]
```diff
@@ -0,0 +1,3 @@
+retomar
+prosseguir
+
```
Comment (augustnmonteiro) [inline vocab/pt-pt/ResumeKeyword.voc:3]: add `play`

File: vocab/pt-pt/PlayKeyword.voc [hunk 5]
```diff
@@ -0,0 +1 @@
+reproduzir
```
Comment (augustnmonteiro) [inline vocab/pt-pt/PlayKeyword.voc:1]: add `play`

**Review by augustnmonteiro — CHANGES_REQUESTED** [review]

**Comment by penrods** [comment]
@augustnmonteiro I'm going to go ahead and merge these -- better than not having them.  You can add your suggestions in a follow-up PR.

**Pull request merged**